i2c/oled_i2c/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+add_executable(oled_i2c
+        oled_i2c.c
+        )
+
+# Pull in our (to be renamed) simple get you started dependencies
```

Comment:
In the BMP280 PR didn't we decide to change this line? :wink:

Reply:
Grrr...